deploy/helm/sumologic/templates/setup/job.yaml

```
@@ -15,6 +15,7 @@ metadata:
   labels:
     app: {{ template "sumologic.labels.app.setup.job" . }}
     {{- include "sumologic.labels.common" . | nindent 4 }}
+    {{- include "sumologic.labels.otelcol.events" . | nindent 8 }}
```

```suggestion
    {{- include "sumologic.labels.otelcol.events" . | nindent 4 }}
```

`events` shouldn't be in `setup`